lib/server.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const Hapi = require('hapi');
+const Hapi   = require('hapi');
 const Config = require('../config');
 const server = new Hapi.Server({
```

Comment:
nit: add new line.

Also, we try and split out third party libraries above internal ones.

```js
'use strict'; 

const Hapi = require('hapi');

const Config = require('../config');

const server = new Hapi.Server( ... );

...

```